Libraries/StyleSheet/processColorArray.js

```diff
@@ -11,8 +11,11 @@
 'use strict';
 
 const processColor = require('./processColor');
+import type {ColorValue, ProcessedColorValue} from './ColorValueTypes';
 
-function processColorArray(colors: ?Array<string>): ?Array<?number> {
```

Comment:
This file has jest tests, can you add some new tests for this behavior at processColorArray-test.js?